external-providers/generic-external-provider/pkg/generic/provider.go

```diff
@@ -57,7 +57,7 @@ func (p *golangProvider) Init(ctx context.Context, log logr.Logger, c provider.I
 	}
 
 	ctx, cancelFunc := context.WithCancel(ctx)
-	log = log.WithValues("provider", "golang")
+	log = log.WithValues("provider", c.ProviderSpecificConfig["name"])
```

Comment:
This option was not mentioned in the documentation

Reply:
Thank you, will add it to the documentation.